pkg/flare/archive_docker_test.go

```diff
@@ -16,8 +16,8 @@ import (
 
 func TestTrimCommand(t *testing.T) {
 	for in, out := range map[string]string{
-		"/pause":                 "/pause",
-		"nginx -g 'daemon off;'": "nginx …",
+		"/pause":                                     "/pause",
```

Comment:
You should update your `go` to version `1.11.5`. It might also be from your editor. If using VIM with `vim-go` simply `:GoUpdateBinaries` will do it.

Reply:
thanks @gbbr :)

Reply:
Same thing happened to me just earlier 😄